Makefile

```diff
@@ -231,5 +237,29 @@ upload-images: images
 version:
 	@echo ${VERSION}
 
+.PHONY: build-cross
+build-cross: LDFLAGS += -extldflags "-static"
+build-cross:
+ifndef HAS_GOX
+	go get -u github.com/mitchellh/gox
+endif
+	CGO_ENABLED=0 gox -parallel=3 -output="_dist/{{.OS}}-{{.Arch}}/{{.Dir}}" -osarch='$(TARGETS)' $(GOFLAGS) $(if $(TAGS),-tags '$(TAGS)',) -ldflags '$(LDFLAGS)' $(GIT_HOST)/$(BASE_DIR)/cmd/openstack-cloud-controller-manager/
```

Comment:
Any reason to not make the number of parallel compiles configurable?

Reply:
Done